### DISCUSSION TITLE
Dashboard

### DISCUSSION THREAD

**Discussion by amberwnaushahi** [ALIYA2Group/Mod20_Segment_2#18] [Ideas]
We could also use a website to show our project:

1- use earth data to show the map from 1978 to date
2- use web scraping and retrieve news articles related to sea ice and climate change and display on website (could be linked to the date that is showing on the google earth map)
3- use web scraping to show animals that are at risk of extinction

**Comment by 735713038455163** [comment]
@maryamt95 completed 1. I am looking into 2- use web scraping and retrieve news articles related to sea ice and climate change and display on website (could be linked to the date that is showing on the google earth map)
3- use web scraping to show animals that are at risk of extinction for mod 3
